### PR TITLE
Fix:Peak element in a mountain array

### PR DESCRIPTION
**Pull request by Shubh2003**
fix:#6524

Added program for peak index in a mountain array in c and c++

### CODE REVIEW

**Review by itzpiyushjindal — APPROVED** [review]
these codes are appropriate

**Comment by Shubh2003** [comment]
@AdiChat please reviews this PR

**Comment by Shubh2003** [comment]
@itzpiyushjindal when this PR will get merged

File: code/algorithm_applications/src/binary_search/peak_element_in_a_mountain_array/peak.dSYM/Contents/Resources/Relocations/x86_64/peak.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
```
Comment (AdiChat) [inline code/algorithm_applications/src/binary_search/peak_element_in_a_mountain_array/peak.dSYM/Contents/Resources/Relocations/x86_64/peak.yml:1]: Remove binary and non-code files.

Only keep code files peak.c, cpp, py.

Comment (Shubh2003) [reply code/algorithm_applications/src/binary_search/peak_element_in_a_mountain_array/peak.dSYM/Contents/Resources/Relocations/x86_64/peak.yml]: done @AdiChat

**Review by winlover32 — APPROVED** [review]

**Review by winlover32 — APPROVED** [review]
code/algorithm_applications/src/binary_search/peak_element_in_a_mountain_array/peak.cpp

**Pull request closed without merging**